__tests__/app.test.js

```diff
@@ -121,3 +121,35 @@ describe("handles all bad paths", () => {
     })
   })
 })
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: should respond with the newly posted comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This is an article that is worth reading"
+    }
+    return request(app)
+    .post("/api/articles/1/comments")
+    .send(newComment)
+    .expect(201)
+    .then(( { body }) => {
+      const { comment } = body
+      expect(comment.comment_id).toBe(19)
+      expect(comment.body).toBe("This is an article that is worth reading")
+      expect(comment.article_id).toBe(1)
+      expect(comment.author).toBe("butter_bridge")
+      expect(comment.votes).toBe(0)
+      expect(comment).toHaveProperty("created_at", expect.any(String))  
+    })
+  })
+  test("400: should respond with a error message when missing required fields", () => {
+    const newComment = {}
+    return request(app)
+    .post("/api/articles/1/comments")
+    .send(newComment)
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Bad request")
+    })
+  })
```

Comment:
There are many more test here that we could implement...

- what if the comment has some extra properties as well as the required ones? I think this could be a happy path test but you'd need to only take what is required off the request body (also see comment in model).
-  what if the article id is invalid 🍓
- what if the article id is not found
- what if the username doesn't exist